lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_CHARGE_ITEM_READ ||= {
+      'resourceType': 'ChargeItem',
+      'id': '157320807',
+      'meta': {
+        'versionId': '3',
+        'lastUpdated': '2019-08-15T14:22:31Z'
+      },
+      'text': {
+        'status': 'generated',
+        'div': '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Charge Item</b></p><p><b>Status</b>: Billable</p>'\
```

Comment:
nit: space between the last ' and \

Reply:
added [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/050ae5f567c5bc377157d50c3e1f7bcc228fd7e0#diff-5cd46ad5f0af8a24217f67b656e77ce3624d6cefec3a18eddf7adc12bd47d35fR15)